Detectors/TPC/base/include/TPCBase/ParameterDetector.h

```diff
@@ -17,59 +17,35 @@
 
 #include <array>
 #include "DataFormatsTPC/Defs.h"
+#include "SimConfig/ConfigurableParam.h"
+#include "SimConfig/ConfigurableParamHelper.h"
 
 namespace o2
 {
 namespace TPC
 {
 
-/// \class ParameterDetector
-
-class ParameterDetector
-{
- public:
-  static ParameterDetector& defaultInstance()
-  {
-    static ParameterDetector param;
-    return param;
-  }
-
-  /// Constructor
-  ParameterDetector();
-
-  /// Destructor
-  ~ParameterDetector() = default;
-
-  /// Set the TPC length
-  /// \param tpclength TPC length [cm]
-  void setTPClength(float tpclength) { mTPClength = tpclength; }
-
-  /// Set the pad capacitance
-  /// \param cpad Pad capacitance [pF]
-  void setPadCapacitance(float cpad) { mPadCapacitance = cpad; }
-
-  /// Set maximum time bin in case of triggered readout mode
-  /// \param Maximum time bin in case of triggered readout mode
-  void setMaxTimeBinTriggered(TimeBin binMax) { mTmaxTriggered = binMax; }
+struct ParameterDetector : public o2::conf::ConfigurableParamHelper<ParameterDetector> {
 
   /// Get the TPC length
   /// \return TPC length [cm]
-  float getTPClength() const { return mTPClength; }
+  float getTPClength() const { return TPClength; }
 
   /// Get the pad capacitance
   /// \return Pad capacitance [pF]
-  float getPadCapacitance() const { return mPadCapacitance; }
+  float getPadCapacitance() const { return PadCapacitance; }
```

Comment:
maybe these functions are not necessary since the param is a `const struct` and the members can be accessed with `.` notation.

Reply:
These functions were removed